### PR TITLE
Directory fixes

### PR DESCRIPTION
**Pull request by jonhadley**

### CODE REVIEW

File: src/StockportWebapp/Views/stockportgov/Directory/Results.cshtml [hunk 1]
```diff
@@ -29,15 +29,15 @@
...
                                 <p>Choose one or more options to filter your results.</p>
 
                                 @if(Model.Directory.SubDirectories.Any())
                                 {
-                                    <h4>Category</h4>
+                                    <h3 class="h-s">Category</h3>
                                     @foreach(var subDirectory in Model.Directory.SubDirectories)
                                     {
-                                        <p><a href="@Url.Action("DirectoryResults", "Directory", new { slug = subDirectory.Slug })">@subDirectory.Title</a></p>
+                                        <p><a alt="" href="@Url.Action("DirectoryResults", "Directory", new { slug = subDirectory.Slug })">@subDirectory.Title</a></p>
```
Comment (elena-secara) [inline src/StockportWebapp/Views/stockportgov/Directory/Results.cshtml:40]: Do **a** tags need/allow **alt** attributes?

File: src/StockportWebapp/Views/stockportgov/Shared/DirectorySearchResult.cshtml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ﻿@model DirectorySearchResultViewModel
 <li class="navcard navcard--white navcard--border-top navcard--force-border-top navcard--width-100">
     <div class="navcard__content">
-        <a href=@Url.ActionLink("DirectoryEntry", "Directory", new { directorySlug=@Model.DirectorySlug, entrySlug=@Model.Entry.Slug }) data-cy="nav-card-link" class="navcard__link navcard__link--border ">
+        <a href=@Url.ActionLink("DirectoryEntry", "Directory", new { directorySlug=@Model.DirectorySlug, entrySlug=@Model.Entry.Slug }, Context.Request.Scheme, null, null) data-cy="nav-card-link" class="navcard__link navcard__link--border ">
```
Comment (elena-secara) [inline src/StockportWebapp/Views/stockportgov/Shared/DirectorySearchResult.cshtml:4]: Is this to fix the "https" requests? Could it be done using @Url.Action?

**Review by elena-secara — CHANGES_REQUESTED** [review]

File: src/StockportWebapp/Views/stockportgov/Shared/DirectorySearchResult.cshtml (hunk 2 above)
Comment (jonhadley) [reply src/StockportWebapp/Views/stockportgov/Shared/DirectorySearchResult.cshtml:4]: Yes , I think you would have to do the same whether it was Url.Action or Url.ActionLink - the only different being Url.Action would also build the <a> tag not just give you the href value?

File: src/StockportWebapp/Views/stockportgov/Directory/Results.cshtml (hunk 1 above)
Comment (jonhadley) [reply src/StockportWebapp/Views/stockportgov/Directory/Results.cshtml]: This was my mistake - I was fixing and issue in the wrong place for some reason!

**Review by elena-secara — APPROVED** [review]

**Pull request merged**